lib/ultralib/include/PR/os_libc.h

```diff
@@ -87,7 +87,7 @@ extern void     bzero(void *, int);
 
 /* Printf */
 
-extern int		sprintf(char *s, const char *fmt, ...);
+extern int sprintf(char* dst, const char* fmt, ...);
```

Comment:
oh, formatter touched ultralib's header. hmmm...
Could you undo this and rename the parameters in the C file to match this header and see if formatter still messes with this file?